cmd/yggd/main.go

```diff
@@ -204,6 +205,14 @@ func main() {
 			}
 		}
 
+		if DefaultConfig.ClientID != "" {
```

Comment:
I'm ok with this, but I would move the validation to the config package, like :

```
if !DefaultConfig.IsValidClientID() { return cli.Exit(...)}
```

Reply:
@subpop @eloycoto 

Ok, I've moved this partially in the config package. But now, it means we disallow the content of `/etc/machine-id` as a client ID! :/